docker-compose.yml

```diff
@@ -90,6 +90,7 @@ services:
       SEQUENCING_DB_USER: "${SEQUENCING_USERNAME}"
       SEQUENCING_DB_PASSWORD: "${SEQUENCING_PASSWORD}"
       SEQUENCING_LOCAL_STORE: /usr/src/app/sequencing_file_store
+      SEQUENCING_LANGUAGE: "SEQN"
```

Comment:
We've changed one docker-compose file, which means:
- [ ] We should change the other ones in the repo
- [ ] We should document the new environment variables in Environment.md

Reply:
Updated [here](https://github.com/NASA-AMMOS/aerie/commit/10086e6fc2a48a667e20f458a73c36fd2e7dc843).